### PR TITLE
fix(clients): read the wired- counter keys wired clients actually report

### PR DESCRIPTION
**Pull request by jpennin5**
## Description

**Problem:** every wired client reports zero traffic — observed live as a
hypervisor moving gigabytes while `get_client_statistics` returned
`tx_bytes: 0`.

**Root cause:** the sta route reports wired clients' traffic as
`wired-tx_bytes`, `wired-rx_bytes`, `wired-tx_packets` and
`wired-rx_packets`; the plain keys are absent or zero for them. Every
client tool read only the plain keys.

**Fix:** the `Client` model fills the plain fields from the `wired-` keys
in a before-validator (covering `list_active_clients`,
`get_client_details` and `search_clients`), and `get_client_statistics`
reads both key families. Wireless clients are unaffected: their plain keys
are present and win.

**Live verification (Network 10.5.67):** a wired hypervisor that
previously reported `tx_bytes: 0` reports its real multi-gigabyte
counters.

## Type of Change

- [x] Bug fix (non-breaking change that fixes an issue)

Output keys are unchanged; wired clients' values become correct.

## Testing

- Fixtures add wired clients with the real hyphenated counter keys; tests
  cover the validator fill, the plain-keys-win rule for wireless, and
  `get_client_statistics` reading both families.
- `pytest tests/unit/` and `pre-commit run --all-files` pass.

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (CHANGELOG.md)
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published (none — stands alone)

## AI Assistance

This PR was created with assistance from Claude Code.

**Human Review Status:** ✅ Reviewed and approved by @jpennin5
**Testing:** Full unit suite passing; before/after counters verified
against a live Network 10.5.67 controller.
**Security Review:** Completed — all fixture values are synthetic or from
documentation ranges.

### CODE REVIEW

File: src/tools/clients.py [hunk 1]
```diff
@@ -90,12 +90,22 @@ async def get_client_statistics(
...
                 # Extract statistics
+                # Wired clients report counters under the "wired-" keys;
+                # the plain keys are absent or zero for them.
                 stats = {
                     "mac": client_mac,
-                    "tx_bytes": client_data.get("tx_bytes", 0),
-                    "rx_bytes": client_data.get("rx_bytes", 0),
-                    "tx_packets": client_data.get("tx_packets", 0),
-                    "rx_packets": client_data.get("rx_packets", 0),
+                    "tx_bytes": client_data.get("tx_bytes")
```
[the remark this answers is not in the document]
Comment (jpennin5) [reply src/tools/clients.py]: Fixed in d376ead: the fallback is now gated on `is_wired is True` with an explicit None/zero check, so a wireless client's legitimate 0 can never be replaced by a stray `wired-` value.

File: src/models/client.py [hunk 2]
```diff
@@ -50,6 +52,22 @@ class Client(BaseModel):
...
+        Wired clients report their counters as ``wired-tx_bytes``,
+        ``wired-rx_bytes``, ``wired-tx_packets`` and ``wired-rx_packets``;
+        the plain keys are absent or zero for them. Without this, every
+        wired client reports no traffic at all.
+        """
+        if isinstance(data, dict):
+            for field in ("tx_bytes", "rx_bytes", "tx_packets", "rx_packets"):
+                if not data.get(field) and data.get(f"wired-{field}") is not None:
+                    data = {**data, field: data[f"wired-{field}"]}
+        return data
```
[the remark this answers is not in the document]
Comment (jpennin5) [reply src/models/client.py]: Fixed in d376ead — the validator fills only when the record reports `is_wired: true`, with `(plain is None or plain == 0)` spelled out. Added a regression test for a wireless zero alongside stray `wired-` keys.

**Comment by enuno** [comment]
### Summary
Request changes. The PR is not yet merge-ready because the current status checks are not clean.

### Critical findings
- Merge state: \ / \.
- Failing check(s): Mermaid Diagram Sync Assistant.
- The changed tooling surface should be revalidated after the failing check is resolved.

### Suggestions
1. Resolve the failing status check(s) and rerun CI.
2. Reconfirm the changed tooling behavior against the intended controller/fixture shapes.
3. Ensure regression tests cover the original failure mode and edge cases introduced here.
4. Update docs if any request/response or error semantics changed.

### Verification checklist for the author
- [ ] Required checks are passing
- [ ] Regression tests cover the changed path
- [ ] Lint / format / type checks pass locally
- [ ] Documentation is updated where user-facing behavior changed
- [ ] No secrets, credentials, or controller-identifying data were introduced

**Comment by jpennin5** [comment]
The failing check is the `Mermaid Diagram Sync Assistant` GitHub App, not a workflow in this repository. Its check-run output on this PR reads: "Webhook Processing Failed — An unexpected error occurred while processing the webhook. Please try pushing a new commit or check your repository configuration." That processing happens on the app's side; pushing new commits to these PRs has re-triggered the check with the same result, so it is not reachable from a fork.

This is the loop documented in #112: the app has errored on every fork PR since late June, fork CI runs sit at `action_required` awaiting maintainer approval, and the daily automated review then reads the app error as a failing required check and requests changes.

Every check that can conclude on a fork PR is green here (both Socket Security checks). The remaining steps are repo-side — approving the CI runs and either repairing/removing the app integration or unrequiring its check. #117 merged with the identical check state, so the merge path exists. Happy to rebase or adjust anything code-side the moment CI can actually run.
